recipes/intake-astro/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.1.1" %}
+{% set name = "intake-astro" %}
+{% set sha256 = "8f2a5f49a10a52cd58770154075de27229242e9ae5c4e95c6a449b285653b90d" %}
+
+package:
+   name: intake-accumulo
+   version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    - jinja2
+  run:
+    - python
+    - intake>=0.3
```

Comment:
Also, doesn't this need a space? `intake >=0.3`

Reply:
added one, but I would have expected the linter to complain about things like that

Reply:
Indeed that should not fail silently :-/